README.md

```diff
@@ -35,15 +37,33 @@ https://discord.com/developers/applications
 
 1. Copy the Link provided in the Scopes panel and paste it into your browser, and invite + approve the bot to the Test Server you created earlier.
 
+
 1. Okay! Now you have an Offline Test Bot in your Test Server. To get it online, you need to navigate to the Bot panel in Discord Developer Portal.
 
+## Authentication, option 1
+
 1. Under Build-A-Bot, click to reveal the token, the copy the token. Go to server.js under the root directory of your cloned repo. Change the following line, as of writing this it is on line 8:
-	```js
-	const token = 'your-token' // await getParameterStore('lucretia-bott-token')
-	``` 
+    ```js
+      const token = 'your-token' // LOCAL ? DISCORD_TOKEN : await getParameterStore('lucretia-bott-token')
```

Comment:
shouldn't this instruction be changed if it can now be included in dotenv?

Reply:
yes i put it in below as option 2, i just didnt want to delete your stuff!

Reply:
LMK if you want me to and i WILL!

Reply:
No one should do it this way. LOL